src/NuGet.Clients/NuGet.VisualStudio.Implementation/SolutionExplorer/Items/AttachedItemPriority.cs

```diff
@@ -23,5 +23,7 @@ internal static class AttachedItemPriority
         public const int CompileTimeAssemblyGroup = 400;
         public const int FrameworkAssemblyGroup = 500;
         public const int ContentFilesGroup = 600;
+        public const int PackageBuildFileGroup = 700;
```

Comment:
There's also `buildTransitive`, are we going to display that?  I guess its sort of an oddity and would just show up under the Build Files?

Reply:
I don't see that as a separate entry on `LockFileTargetLibrary` (there's only `Build` and `BuildMultiTargeting`). Perhaps they're being placed in one of those collections. Do you know of a package that has `buildTransitive` content I can test with?

While you're thinking about this, are there other items in `LockFileTargetLibrary` that'd be useful to surface in the UI?

https://github.com/NuGet/NuGet.Client/blob/ba01cc633f88fb2dd1bfcbcdd859fb89525a56eb/src/NuGet.Core/NuGet.ProjectModel/LockFile/LockFileTargetLibrary.cs#L37-L39

Reply:
Yeah sorry I was thinking more from the perspective of a package having these assets, but as you said this is more about what's in the assets file.  I don't think we differentiate that, it would just be a build asset that comes from a transitive package reference.  I don't know of a package off-hand that has it.  Not sure if any other assets should show up in the UI, I'll let others comment on that one.

Reply:
buildTransitive is included under build. 

Are FrameworkReferences displayed in some way? 

That's the only thing that I'd say would be valuable.

Reply:
> Are FrameworkReferences displayed in some way?

We show the project's framework references in a group alongside packages. What would be the significance of a package's framework reference?

Reply:
Framework references are transitive, so a package can specify a framework reference and it would be automatically included in the project. 

Knowing where the framework reference is originally coming from might be helpful.